test/fixtures/ruby_gems_users.yml

```diff
@@ -0,0 +1,16 @@
+
+one:
+  user_id: 1
+  ruby_gem_id: 1
+
+two:
+  user_id: 1
+  ruby_gem_id: 2
+
+three:
+  user_id: 1
+  ruby_gem_id: 3
+
+four:
+  user_id: 3
+  ruby_gem_id: 4
```

Comment:
@sebasoga The idea was to populate the testing database with some sample associations between users and ruby gems to test the method ´#sync_gems´ for an user in the following cases:

* When the user remove a gem from RubyGems.org make sure the association was removed
* When the user adds a gem to RubyGems.org make sure a new association was made